.dependency_license

```diff
@@ -111,6 +111,8 @@ traffic_portal/app/src/assets/js/chartjs/angular-chart\..*, BSD
 traffic_portal/app/src/assets/css/jsonformatter\..*, Apache
 traffic_portal/app/src/assets/js/jsonformatter\..*, Apache
 traffic_portal/app/src/assets/js/fast-json-patch\..*, MIT
+traffic_portal/app/src/assets/css/angular-ui-tree\..*, MIT
+traffic_portal/app/src/assets/js/angular-ui-tree\..*, MIT
```

Comment:
These regexes need to be adjusted to cover:
- `traffic_portal/app/src/assets/css/angular-ui-tree_2.22.6.css`
- `traffic_portal/app/src/assets/js/angular-ui-tree-min_2.22.6.js`

Reply:
PR submitted to fix this